sriov/tests/SR_IOV_MultipleVFCreation_withMAC_DPDK/test_SR_IOV_MultipleVFCreation_withMAC_DPDK.py

```diff
@@ -0,0 +1,48 @@
+from sriov.common.utils import (
+    execute_and_assert,
+    create_vfs,
+    set_pipefail,
+    get_pci_address,
+    bind_driver,
+)
+
+
+# Use pytest --iteration to adjust the execution_number parameter for desired amount
+# of repeated tests
+def test_SRIOVMultipleVFCreationwithMACDPDK(dut, settings, testdata, execution_number):
+    """Test and ensure that VFs provision with MTU functions as intended
+
+    Args:
+        dut:              ssh connection obj
+        settings:         settings obj
+        testdata:         testdata obj
+        execution_number: execution_number parameter
+    """
+    pf = list(testdata.pfs.keys())[0]
+    base_mac = "0x0000000000"
+
+    assert set_pipefail(dut)
+
+    max_vfs_cmd = ["cat " + testdata.pf_net_paths[pf] + "/sriov_totalvfs"]
+    outs, errs = execute_and_assert(dut, max_vfs_cmd, 0)
+    max_vfs = outs[0][0].strip()
+
+    assert create_vfs(dut, testdata.pfs[pf]["name"], int(max_vfs))
+
+    for i in range(int(max_vfs)):
+        base_mac = "{:012X}".format(int(base_mac, 16) + 1)
+        new_mac = ":".join(
+            base_mac[i] + base_mac[i + 1] for i in range(0, len(base_mac), 2)
+        )
+        steps = [
+            "ip link set "
+            + testdata.pfs[pf]["name"]
+            + " vf "
+            + str(i)
+            + " mac "
+            + new_mac
+        ]
+        execute_and_assert(dut, steps, 0)
+
+        vf_pci = get_pci_address(dut, testdata.pfs[pf]["name"] + "v" + str(i))
+        assert bind_driver(dut, vf_pci, "vfio-pci")
```

Comment:
we need to add a time bound to this step; one way to do it is to calculate the time spend on this step, if is it over say 3 second * max_vf, then fail it; another way to do it is for each step in bind_driver, use 1 second timeout in stead of the default execute timeout when executing this test case

Reply:
Added timeout in [675db54](https://github.com/redhat-partner-solutions/rhel-sriov-test/pull/87/commits/675db54d57fee9c07df4a09b77bb1627a2be81ed)